Makefile

```diff
@@ -4,18 +4,12 @@ default:
 	@echo "EKL: Elastic Search & Kibana & Logstash"
 	@echo "make [<ops>|<dev>]"
 	@echo "make ops: runs EKL in a local dev environment"
-	@echo "make dev: opens erlang shell"
-	@echo "make test: test install script"
+	@echo "make test: runs the test erlang application"
 
 ops:
-	cd ops && \
-	docker-compose build && \
-	docker-compose up
-
-ops_reset:
-	cd ops && docker-compose down
-dev:
-	rebar3 release && rebar3 shell
+	cd ops/ensure_kibana_configuration && \
+	 docker-compose up --build --force-recreate -d
```

Comment:
remove space

Reply:
space was added because it clarifies that is a continuation of the command above. Isn't necessary is this cases?

Reply:
oh let me check i thought it was the end of the file, probably my mistake